lib/bot.js

```diff
@@ -152,6 +153,11 @@ class Bot {
       return;
     }
 
+    if (message.user === 'USLACKBOT' && this.muteSlackbot) {
```

Comment:
Squash and force-push would be fine :+1:. You probably know already, but you can amend changes to the previous commit by adding the files and doing `git commit --amend --no-edit` (followed by a force-push).